src/utils/validators.js

```diff
@@ -16,7 +16,7 @@ const PERMALINK_REGEX = "^((/([a-zA-Z0-9]+-)*[a-zA-Z0-9]+)+)/?$"
 export const URL_REGEX_PREFIX = "^(https://)?(www.)?("
 export const URL_REGEX_SUFFIX = ".com/)([a-zA-Z0-9_-]+([/.])?)+$"
 export const YOUTUBE_REGEX = ".com/(@)?)([a-zA-Z0-9_-]+([/.])?)+$"
-export const TELEGRAM_REGEX = "telegram|t).me/([a-zA-Z0-9_-]+([/.])?)+$"
+export const TELEGRAM_REGEX = "telegram|t).me/\\+?([a-zA-Z0-9_-]+([/.])?)+$"
```

Comment:
shouldn't it be `\/`?

Reply:
Hmm no, we want to add the optional `+` at the start after the `/`, the `/` doesn't need to be escaped here but the `+` need.

Reply:
ok clarified offline - this is a string, so we need to escape the escape character